src/server/middleware/auth.ts

```diff
@@ -248,6 +248,16 @@ const handleWebsocketAuth = async (
     // Set as a header for `getUsers` to parse the token.
     req.headers.authorization = `Bearer ${jwt}`;
     const user = await getUser(req);
+
+    const isIpInAllowlist = await checkIpInAllowlist(req);
+    if (!isIpInAllowlist) {
+      return {
+        isAuthed: false,
+        error:
+          "Unauthorized IP Address. See: https://thirdweb.com/dashboard/engine",
```

Comment:
We should link to the docs page for ip allowlist. Can we link to https://portal.thirdweb.com/engine/features/ip-allowlist and add docs there?

Devs would be confused just going to the Engine dashboard.